test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -1,10 +1,12 @@
 import { createHash, randomBytes } from "crypto"
 
-import { Wallets, Lightning } from "@app"
+import { Wallets, Lightning, Payments } from "@app"
+import { getMidPriceRatio } from "@app/payments/helpers"
```

Comment:
How should we handle a case like this where technically the methods aren't needed outside of the `payments` package but I'm importing them in tests to compare against. Should we also expose these in the top level `payments/index`, or is this direct import ok?